components/Tabs2/src/MDCTabBarView.h

```diff
@@ -19,4 +19,10 @@
  */
 __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScrollView
 
+/** The set of items displayed in the Tab bar. */
+@property(nonnull, nonatomic, copy) NSArray<UITabBarItem *> *items;
```

Comment:
UITabBar's items can be `nullable`; do we want to follow suit here as well?

Reply:
I think a good argument for making this nonnull is that it makes the Swift code a bit easier to work with, so I'm supportive of this deviation in that regard. I'd love y'all's thoughts on this.

If we do decide to keep this as nonnull, please add a test verifying that `items` is not nil after initialization:

```objc
// Given
MDCTabBarView *tabBarView = [[MDCTabBarView alloc] init];

// Then
XCTAssertNotNil(tabBarView.items);
```

Reply:
+1 I think `nullable` is fine to match UITabBar.

Reply:
MDCTabBar's property is `nonnull`, so making it `nullable` will certainly make Swift migration more expensive. I'm not sure there's much of a benefit to a `nil` array versus an empty one.  I feel like I'm waffling, but I don't feel strongly one way or another.

Reply:
I say we skate where the puck is going on this one and make the API more friendly for Swift folks. But please add the test (+ you'll find that you're missing an initializer :) ).

Reply:
Thanks! Tests and initializer added :)